overviewer_core/src/block_class.c

```diff
@@ -0,0 +1,190 @@
+/* 
+ * This file is part of the Minecraft Overviewer.
+ *
+ * Minecraft Overviewer is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Minecraft Overviewer is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with the Overviewer.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "block_class.h"
+#include "utils.h"
+
+#if defined(__i386__) || defined(__x86_64__)
+#include <immintrin.h>
+#endif
+
+bool block_class_is_subset(
+	mc_block_t block,
+	const mc_block_t block_class[],
+	size_t block_class_len
+)
+{
```

Comment:
Please put the closing parenthesis and the opening brace on the same line as the last parameter of a function, unless it leads to misleading indentation.